Clarify why we are requiring at least 3 organizations in the Impact project leadership (#1402)
openjs-foundation/cross-project-council: PROJECT_PROGRESSION.md

#### Definition

The Impact Stage is for projects that have reached their growth goals and are now on a sustaining cycle of development, maintenance, and long-term support. Impact Stage projects are used commonly in enterprise production environments and have large, well-established project communities.

> Impact projects offer long-term stability guarantees, and therefore the governing body of Impact projects are expected to have representatives from at least three organizations.

Reviewer: ```suggestion
Impact projects offer long-term stability guarantees, and therefore the governing bodies of Impact projects are expected to have representatives from at least three organizations.
```

#### Definition

The Impact Stage is for projects that have reached their growth goals and are now on a sustaining cycle of development, maintenance, and long-term support. Impact Stage projects are used commonly in enterprise production environments and have large, well-established project communities.    

> Impact projects offer long-term stability guarantees, and therefore the governing body of Impact projects are expected to have representatives from at least threee organizations.

Reviewer 2: ```suggestion
Impact projects offer long-term stability guarantees, and therefore the governing body of Impact projects are expected to have representatives from at least three organizations.
```

To graduate from At Large stage or for a new project to join as an Impact project, a project must meet the following criteria:

 * Demonstrate that it is being used successfully in production by at least two independent end users which, in the CPC’s judgement, are of adequate quality and scope.

>  * Have a defined governing body of at least 5 or more members (owners and core maintainers), of which no more than 1/3 is affiliated with the same employer. In the case there are 5 governing members, 2 may be from the same employer. This is to ensure that no single organization can control the project.

Reviewer 3: Totally optional suggestion, but something "excessive influence" might be less loaded than "control":

```suggestion
 * Have a defined governing body of at least 5 or more members (owners and core maintainers), of which no more than 1/3 is affiliated with the same employer. In the case there are 5 governing members, 2 may be from the same employer. This limitation is to ensure broad representation and to prevent a single organization from having excessive influence on project decision-making.
```

While that less loaded, it might also be less clear, so...up to you. Just offering it up as a possibility.

>  * Have a defined governing body of at least 5 or more members (owners and core maintainers), of which no more than 1/3 is affiliated with the same employer. In the case there are 5 governing members, 2 may be from the same employer. This limitation is to ensure broad representation and to prevent a single organization from having excessive influence on project decision-making.

Reviewer: ```suggestion
 * Have a defined governing body of at least 5 or more members (owners and core maintainers), of which no more than 1/3 is affiliated with the same employer. In the case there are 5 governing members, 2 may be from the same employer. This criterion is to ensure broad representation and to prevent a single organization from having excessive influence on project decision-making.
```
